src/libraries/grommet/components.js

```diff
@@ -1229,6 +1231,9 @@ export const components = {
     designProperties: {
       dataPath: '',
     },
+    properties: {
+      kind: '',
```

Comment:
Can we do what we do with Button.kind, where we pick values from the theme so the user can then select from those options? Perhaps `kind: ['-theme-'],`?

Reply:
Hmm I tried setting `kind: ['-theme-']` but now I no longer see `kind` as an option under Form properties

Reply:
anything in the console?